app/CMakeLists.txt

```diff
@@ -201,6 +201,8 @@ set(app_desktop_SRCS
     src/heartbeat/date_provider.cc
     src/heartbeat/heartbeat_storage_desktop.cc
     src/heartbeat/heartbeat_controller_desktop.cc
+    rest/gzipheader.cc
```

Comment:
I'm not very confident about the changes to this file. I know that controller_desktop depends on /rest/zlibwrapper which depends on /rest/gzipheader and zlib itself, and so I imitated the contents of the /rest/CmakeLists file in a way that caused my build to succeed.

Reply:
I updated this file to only include/link zlib source_dir and zlibstatic for desktop (non-ios and non-android)